x-pack/packages/kbn-elastic-assistant-common/README.md

```diff
@@ -18,3 +18,22 @@ To (interactively) run unit tests with code coverage, run the following command:
 ```sh
 cd $KIBANA_HOME && node scripts/jest --watch x-pack/packages/kbn-elastic-assistant-common --coverage
 ```
+
+## OpenAPI Codegen
+
+Implemented using the new OpenAPI codegen and bundle packages:
+* Includes OpenAPI codegen script and CI action as detailed in: https://github.com/elastic/kibana/pull/166269
+* Includes OpenAPI docs bundling script as detailed in: https://github.com/elastic/kibana/pull/171526
+
+To run codegen/bundling locally, cd to `x-pack/packages/kbn-elastic-assistant-common/` and run any of the following commands:
+
+```bash
+yarn openapi:generate
+yarn openapi:generate:debug
+yarn openapi:bundle
+```
+
+Codegen is configured to run on CI by means of the `.buildkite/scripts/steps/code_generation/elastic_assistant_codegen.sh` script, which is run as part of the `checks` pipeline, and is registered in `.buildkite/scripts/steps/checks.sh`.
+
+> [!NOTE]
+> At the moment `yarn openapi:bundle` will output an empty bundled schema since `get_capabilities_route` is an internal route, this is to be expected. Also, if you don't see the file in your IDE, it's probably because `target` directories are ignored, so you may need to manually find/open the bundled schema at its target location: `x-pack/packages/kbn-elastic-assistant-common/target/openapi/elastic_assistant.bundled.schema.yaml`
```

Comment:
Would it be more effective to add this note to the `openapi:bundle` script's output, instead of here? I don't have a sense of when/how often users would encounter this issue, but if we really want to communicate this to the caller, the script seems like a better place.